base/c.jl

```diff
@@ -110,8 +106,17 @@ end
 # symbols are guaranteed not to contain embedded NUL
 convert(::Type{Cstring}, s::Symbol) = Cstring(unsafe_convert(Ptr{Cchar}, s))
 
-# FIXME: this should be handled by implicit conversion to Cwstring, but good luck with that
 if is_windows()
+"""
+    Base.cwstring(s)
+
+Converts a string `s` to a NUL-terminated `Vector{Cwchar_t}`, suitable for passing to C
+functions expecting a `Ptr{Cwchar_t}`. The main advantage of using this over the implicit
+conversion provided by `Cwstring` is if the function is called multiple times with the
+same argument.
+
+This is only available on Windows.
```

Comment:
Looks like @Keno is right: `Cwstring` appears in base/exports.jl but not `cwstring`.